lib/runner.js

```diff
@@ -2,13 +2,15 @@
 
 const { EventEmitter } = require('events');
 const { ConciseReporter } = require('./report');
+const { TapReporter } = require('./report');
 
 class Runner extends EventEmitter {
 
   constructor(options) {
     super();
     this.options = Object.assign({ runTodo: false }, options);
-    this.reporter = new ConciseReporter();
+    options = Object.assign({ reporter: new TapReporter() }, options);
```

Comment:
This line is not needed since you already have `ConciseReporter` as your default.
```suggestion
```